debugger.hpp

```diff
@@ -3,11 +3,34 @@
 
 #include <string>
 #include <sstream>
+#include "node.hpp"
 #include "ast_fwd_decl.hpp"
 
 using namespace std;
 using namespace Sass;
 
+/*
```

Comment:
Would like to keep this since it is working, but I think I got some problem within a certain source file, so I commented it out (since it is for debugging I vote to leave it in at least for now).